### ISSUE TITLE
support "undefined" in templates

### ISSUE THREAD

**Issue by staeke** [angular/angular#7625]
Type: **bug?** 

I argue it's common and useful thing to be able to compare with ```undefined```. However, currently it seems the value ```undefined``` isn't supported in templates (Angular 2 beta 9). See this Plunker for example (http://plnkr.co/edit/h1wE4sBb6vYVTVfieNuH?p=preview)

Basically it comes down to that
```
{{myUndefinedVar === undefined}}
```
...produces the output
```
false
```

Now that may sound like a feature request. But comparing to an undefined variable "works". In other words
```
{{myUndefinedVar === someOtherNonExistingVar}}
```
...produces the output
```
true
```

This is so confusing that I'd call it a bug.

**Comment by mirkonasato** [comment]
And (as you show in your plunk) `{{nullValue === null}}` shows `true`.
Also `{{undefinedValue == undefined}}` shows `true`.

**Comment by Andrey-Pavlov** [comment]
I dont know why `{{myUndefinedVar === undefined}}` not works but `{{myUndefinedVar === UNDEFINED}}` is fine.

**Comment by pkozlowski-opensource** [comment]
Fixed in master via b4613ab2d26b0546699bad1afd7c9441b324382f
